identity_iota_core/src/document/iota_document_metadata.rs

```diff
@@ -21,6 +22,10 @@ pub struct IotaDocumentMetadata {
   pub updated: Option<Timestamp>,
   #[serde(skip_serializing_if = "Option::is_none")]
   pub deactivated: Option<bool>,
+  #[serde(rename = "governorAddress", skip_serializing_if = "Option::is_none")]
+  pub governor_address: Option<Address>,
+  #[serde(rename = "stateControllerAddress", skip_serializing_if = "Option::is_none")]
+  pub state_controller_address: Option<Address>,
```

Comment:
I think these fields are included when publishing now (if you resolve the DID then publish an update). We should avoid bloating the payload unnecessarily since the information is in the Alias Output itself, so unset these when packing or perhaps use a different metadata struct for the packed representation without these fields.